buckets/redis/bucket_factory.go

```diff
@@ -57,6 +60,7 @@ accumulatedTokens = accumulatedTokens - accumulatedTokensUsed
 if (tokensNextAvailableNanos - currentTimeNanos > maxDebtNanos) or (waitTime > 0 and waitTime > maxWaitTime) then
 	waitTime = -1
 else
+	redis.replicate_commands()
```

Comment:
Redis doesn't allow non-deterministic functions unless we use replicating commands instead of scripts
https://redis.io/commands/EVAL#replicating-commands-instead-of-scripts

Reply:
maybe put that information in a comment above `luaScript`?